tools/hangar/__snapshots__/test_corpus/redis.w_test_sim.md

```diff
@@ -1,9 +0,0 @@
-# [redis.w](../../../../examples/tests/valid/redis.w) | test | sim
```

Comment:
Redis tests are actually skipped in ci, but I am surprised it skipped and deleted it locally. I would say undo this change for now

Reply:
<img width="617" alt="image" src="https://user-images.githubusercontent.com/39455181/234221683-273564e7-c06e-41f6-b2e2-a6989bc4d99e.png">
seems to be skipped on local too, I'll take a look :)

Reply:
This code is responsible for that:
https://github.com/winglang/wing/blob/746c0f17a4362b77b9c1095f7f28bc3a8ee238b7/tools/hangar/src/generate_tests.ts#L16-L31
here is the meta-comment:
<img width="364" alt="image" src="https://user-images.githubusercontent.com/39455181/234222919-d071a21e-ccbd-4120-817b-633683246ef0.png">
and my machine is the one to skip:
<img width="271" alt="image" src="https://user-images.githubusercontent.com/39455181/234223135-b2eb9d8d-f8cf-41dc-a8f1-d641532cf489.png">

+ if a contributor will run it they might have the other skipped platform, and it'll happen again...
What do you think will be the best way to solve it @MarkMcCulloh?

Reply:
Ah, I think I intended to only skip on CI. I think it would be sufficient to add a check only to respect skipPlatforms when `process.env.CI` is `true`

Reply:
perfect! thanks :) 
we already have `process.env.CI` right?

Reply:
Done :)